src/controllers/articles.js

```diff
@@ -11,6 +11,8 @@ exports.get = (req, res, next) => {
         style: ['articles', 'header', 'footer', 'public'],
         javascript: ['hamburger'],
         articlesArray: result.rows,
+        login: (res.locals.unlockCookie === null),
```

Comment:
so if there is no cookie then this would evaluate to true!
so hw would be marked as login without having a cookie 
shouldn't be !==null